### PR TITLE
Capitalize command name and fix markdown issue in README

### PR DESCRIPTION
**Pull request by vulpivia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ Search for `Capitalize` in the _Extensions_ section.
 
 ## Usage
 
-Select the text you want to capitalize and execute the `Capitalize title` command via the command palette.
+Select the text you want to capitalize and execute the `Capitalize Title` command via the command palette.
 
 ## Configuration
 
-#### `capitalize.special`
+**`capitalize.special`**
```
Comment (mrwest808) [inline README.md]:
```suggestion
#### `capitalize.special`
```

Even though there's only one configuration option now, I'd rather have them as headings to be able link to them directly.

Comment (vulpivia) [reply README.md]: Okay, that makes sense :+1:

**Comment by mrwest808** [comment]
Thanks for contributing, I hadn't noticed that command names in the command palette were capitalized. Apart from the comment/suggestion I left this looks good!

**Comment by vulpivia** [comment]
Most of them are capitalized, some aren't. It's actually a bit chaotic at the moment.

commands

**Pull request merged**